bindings.c

```diff
@@ -1909,11 +1909,17 @@ int cg_open(const char *path, struct fuse_file_info *fi)
 
 int cg_access(const char *path, int mode)
 {
+	int ret;
 	const char *cgroup;
-	char *last = NULL, *path1, *path2, * cgdir = NULL, *controller;
+	char *path1, *path2, *controller;
+	char *last = NULL, *cgdir = NULL;
 	struct cgfs_files *k = NULL;
 	struct fuse_context *fc = fuse_get_context();
-	int ret;
+
+	if (strcmp(path, "/cgroup") == 0) {
+		ret = 0;
```

Comment:
I think you should check the mode for write access and return an error.  Is there a reason not to?